lib/public/Security/CSRF/ICsrfValidator.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OCP\Security\CSRF;
```

Comment:
```suggestion
namespace OCP\Security\Csrf;
```

maybe

https://stackoverflow.com/questions/2236807/java-naming-convention-with-acronyms

Reply:
Using `Csrf` where possible, renaming the namespace with the existing classes is too risky :worried: